Installers/Mac/Setup/Contents/MacOS/minima

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+# Constants
+JAVA_MINIMUM=11
+APP_JAR="minima.jar"
+APP_NAME="Minima"
+VM_ARGS="-rpcenable true"
+MINIMA_FOLDER=$HOME/.minima
+
+# Set the working directory
+DIR=$(cd "$(dirname "$0")"; pwd)
+
+# Error message for NO JAVA dialog
+ERROR_TITLE="Cannot launch $APP_NAME"
+ERROR_MSG="$APP_NAME requires Java version $JAVA_MINIMUM or later to run."
+DOWNLOAD_URL="https://www.oracle.com/uk/java/technologies/javase/jdk11-archive-downloads.html"
+
+# Is Java installed?
+if type -p java; then
+    _java="java"
+elif [[ -n "$JAVA_HOME" ]] && [[ -x "$JAVA_HOME/bin/java" ]]; then
+    _java="$JAVA_HOME/bin/java"
+else
+    osascript \
+	-e "set question to display dialog \"$ERROR_MSG\" with title \"$ERROR_TITLE\" buttons {\"Cancel\", \"Download\"} default button 2" \
+	-e "if button returned of question is equal to \"Download\" then open location \"$DOWNLOAD_URL\""
+	echo "$ERROR_TITLE"
+	echo "$ERROR_MSG"
+	exit 1
+fi
+
+if [ ! -d "$HOME/.minima" ]; then
+	mkdir $MINIMA_FOLDER
+	chmod u+x .minima
+fi
+
+if [ ! -f "$MINIMA_FOLDER/minima.sh" ]; then
+	cd $MINIMA_FOLDER
+
+	cp $DIR/../Resources/minima.sh $MINIMA_FOLDER
+	chmod +x $MINIMA_FOLDER/minima.sh
+	
+	echo 'alias minima='$MINIMA_FOLDER'/minima.sh' >>~/.bash_profile
```

Comment:
is it possible to put this on the binary path instead?
not common practice to setup aliases in people's profiles I dont think
would make uninstall awkward too

Reply:
I've updated installer to open in a terminal window so this isn't required